ingestion/transformers/assign_cadet_databases.py

```diff
@@ -14,9 +14,13 @@
 from ingestion.config import ENV, INSTANCE, PLATFORM
 from ingestion.dbt_manifest_utils import (
     get_cadet_manifest,
-    validate_fqn,
     parse_database_and_table_names,
+    validate_fqn,
 )
+from ingestion.utils import st_time
+
+logging.basicConfig()
+logging.getLogger(__name__).setLevel(logging.DEBUG)
```

Comment:
What is this `setLevel` for? It looks like nothing is logged to this logger at the moment, as the return value of `logging.getLogger` is discarded. Was the intention to set a `logger` variable and use it? Or get the root logger?

Reply:
> Was the intention to set a logger variable and use it?

This